wa/framework/execution.py

```diff
@@ -486,6 +486,9 @@ def run_next_job(self, context):
             if self.context.reboot_policy.reboot_on_each_job:
                 self.logger.debug('Rebooting on new job.')
                 self.context.tm.target.reboot()
+            elif self.context.reboot_policy.reboot_on_each_spec and context.spec_changed:
+                self.logger.debug('Rebooting on new spec.')
```

Comment:
These messages should be upgraded to info, now that I think about it.

Reply:
Updated and added logging for initial reboot.